tailcfg/tailcfg.go

```diff
@@ -2307,6 +2307,11 @@ const (
 	// Added 2024-05-29 in Tailscale version 1.68.
 	NodeAttrSSHBehaviorV1 NodeCapability = "ssh-behavior-v1"
 
+	// NodeAttrSSHBehaviorV2 forces SSH to use the V2 behavior (use su, run SFTP in child process).
+	// This overrides NodeAttrSSHBehaviorV1 if set.
+	// Added 2024-08-06 in Tailscale version 1.72.
```

Comment:
is there a URL you can put in here for more info on what "v2" means?

Reply:
Not a URL, but I added a note about what to look for in incubator.go to distinguish V1 and V2 behavior.